source/administration/release-definitions.rst

```diff
@@ -1,19 +1,10 @@
 Release Definitions
 ===================
 
-Mattermost ships with a new version on the 16th of each month in `binary form <https://docs.mattermost.com/administration/upgrade.html>`_.
-
-For the past few years, Mattermost used a monthly “tick-tock” alternating release cycle. A tick-tock cycle refers to even-numbered releases (e.g., 5.26) containing new features, and odd-numbered releases (e.g., 5.27) containing only bug fixes and performance improvements. As our product and team continue to evolve, we're moving away from this alternating release cycle in favor of a general monthly release.
-
-Feature Release
-----------------
+General monthly release
```

Comment:
I think this is a bigger effort than just updating the release definitions doc since there are multiple places where we refer to or link to "Feature Release", e.g.

 - [Release Lifecycle doc](https://docs.mattermost.com/administration/release-lifecycle.html)
 - [ESR doc](https://docs.mattermost.com/administration/extended-support-release.html#what-are-the-current-supported-extended-support-release-versions)
 - [Handbook release process doc](https://handbook.mattermost.com/operations/research-and-development/product/release-process/feature-release)
 - [Handbook how to write feature release announcements](https://handbook.mattermost.com/operations/research-and-development/product/how-to-guides-for-product/release-announcement)
 - Links from the changelog to this release definitions doc
 - Release blog posts
 - Potentially other places across marketing/docs/etc

Reply:
I think this particular page should only reflect the current state of the release definition. But i am happy to go through the first 4 bullets on your list and change where i would change it. Just let me know.

Reply:
I think it's fine if you want to open PRs for those and we can review with PMs too.